m/main_glider_data_processing_dt.m

```diff
@@ -589,143 +653,160 @@
+% $$$           if isfield(public_figures_local, figure_name)
+% $$$             disp(['Warning: figure ' figure_name ' appears to be duplicated.']);
+% $$$           else
+% $$$             public_figures_local.(figure_name) = figure_output.(figure_name);
+% $$$           end
+% $$$         end
+% $$$       end
+% $$$     end
+% $$$     public_figure_name_list = fieldnames(public_figures_local);
+% $$$     if ~isempty(public_figure_name_list)
+% $$$       [status, attrout] = fileattrib(public_figure_dir);
+% $$$       if ~status
+% $$$         [status, message] = mkdir(public_figure_dir);
+% $$$       elseif ~attrout.directory
+% $$$         status = false;
+% $$$         message = 'not a directory';
+% $$$       end
+% $$$       if status
+% $$$         for public_figure_name_idx = 1:numel(public_figure_name_list)
+% $$$           public_figure_name = public_figure_name_list{public_figure_name_idx};
+% $$$           figure_local = public_figures_local.(public_figure_name);
+% $$$           figure_public = figure_local;
+% $$$           figure_public.url = ...
+% $$$             [public_figure_baseurl '/' ...
+% $$$              figure_public.filename '.' figure_public.format];
+% $$$           figure_public.dirname = public_figure_dir;
+% $$$           figure_public.fullfile = ...
+% $$$             fullfile(figure_public.dirname, ...
+% $$$                      [figure_public.filename '.' figure_public.format]);
+% $$$           [success, message] = ...
+% $$$             copyfile(figure_local.fullfile, figure_public.fullfile);
+% $$$           if success
+% $$$             public_figures.(public_figure_name) = figure_public;
+% $$$             disp(['Public figure ' public_figure_name ' succesfully copied.']);
+% $$$           else
+% $$$             disp(['Error creating public copy of figure ' ...
+% $$$                   public_figure_name ': ' figure_public.fullfile '.']);
+% $$$             disp(message);
+% $$$           end
+% $$$         end
+% $$$       else
+% $$$         disp(['Error creating public figure directory ' public_figure_dir ':']);
+% $$$         disp(message);
+% $$$       end
+% $$$     end
+% $$$     % Write the figure information to the JSON service file.
+% $$$     if isfield(config.paths_public, 'figure_info') ...
+% $$$         && ~isempty(config.paths_public.figure_info)
+% $$$       disp('Generating figure information service file...');
+% $$$       public_figure_info_file = ...
+% $$$         strfstruct(config.paths_public.figure_info, deployment);
+% $$$       try
+% $$$         savejson(public_figures, public_figure_info_file);
+% $$$         disp(['Figure information service file successfully generated: ' ...
+% $$$               public_figure_info_file]);
+% $$$       catch exception
+% $$$         disp(['Error creating figure information service file ' ...
+% $$$               public_figure_info_file ':']);
+% $$$         disp(message);
+% $$$       end
+% $$$     end
+% $$$   end
```

Comment:
The creation of public copies of the figures can also be disabled in `configDTPathsPublic`